types/smtp-server/index.d.ts

```diff
@@ -3,6 +3,7 @@
 // Definitions by: markisme <https://github.com/markisme>
 //                 taisiias <https://github.com/Taisiias>
 //                 Piotr Roszatycki <https://github.com/dex4er>
+//                 OhMySMTP <https://github.com/ohmysmtp>
```

Comment:
Adding an organization here doesn’t really do anything. If you want to be included as an owner yourself, you need to add your own account.

Reply:
Thanks, updated